src/seedu/addressbook/AddressBook.java

```diff
@@ -208,7 +208,46 @@
 
     public static void main(String[] args) {
         showWelcomeMessage();
+        
+        /* original code before SLAP reduction
         processProgramArgs(args);
+        */
+
+        /** T2A3 Exercise 1
+         * After SLAP reduction
+         */
```

Comment:
Comments should be about the code only. It should still make sense to any future developer reading the comment. If you want to tell something to the reviewer/tutor (but not future developers of the code), use a GitHub comment instead (like this comment I've added).